packages/fxa-profile-server/lib/img-workers.js

```diff
@@ -15,22 +15,47 @@ const WORKER_URL = config.get('worker.url');
 const ACCEPT_ENCODING_HEADER = 'identity';
 
 exports.upload = function upload(id, payload, headers) {
-  return new P(function (resolve, reject) {
+  return new P(function(resolve, reject) {
     var url = WORKER_URL + '/a/' + hex(id);
 
+    var header_excludes = config.get('worker.headers_exclude');
+    var header_includes = config.get('worker.headers_include');
+
+    worker_headers = {};
+    if (header_includes.length) {
+      // do not pass-though any HTTP headers except these
+      for (header in headers) {
+        if (header_includes.includes(header.toLowerCase())) {
+          worker_headers[header] = headers[header];
+        }
+      }
+    } else if (header_excludes.length) {
+      // pass-through all headers except these
+      for (header in headers) {
```

Comment:
`for (const .... of ...)`

Reply:
Hm:
```
TypeError: headers is not iterable at [...]
```

Reply:
Wow, sorry, I thought those were the configured array lists.  My fault for not reading carefully. :(  `for...in` is correct here.

Reply:
...but `for (const ... in ...)` works :) I am pretty new to node.js